packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -149,6 +151,31 @@ export const ContractType = (() => {
                     delete available_categories[key];
                 }
             });
+
+            non_available_categories = {};
```

Comment:
isn't it possible to move this out to a hook or something?
just want to make sure that we are not adding new stuff to the stores so we get rid of it evantually.